addons/html_builder/static/src/snippets/snippet_service.js

```diff
@@ -92,17 +93,17 @@ export class SnippetModel extends Reactive {
             body: markup(
                 `${escape(bodyText)}\n<a href="${linkUrl}" target="_blank">${escape(linkText)}</a>`
             ),
-            confirm: async () => {
+            confirm: () => {
                 try {
-                    await this.orm.call("ir.module.module", "button_immediate_install", [
-                        [Number(snippet.moduleId)],
-                    ]);
-                    // TODO Need to Reload webclient
-                    // this._onSaveRequest({
-                    //     data: {
-                    //         reloadWebClient: true,
-                    //     },
-                    // });
+                    this.uiService.block();
+                    const install = async () => {
+                        await this.orm.call("ir.module.module", "button_immediate_install", [
+                            [Number(snippet.moduleId)],
+                        ]);
+                        await this.reload();
+                        this.uiService.unblock();
+                    };
+                    install();
```

Comment:
if your rpc has a crash you never unblock

Reply:
We need to reload the full page if we want to have the new js file. (You don t have to much issue, because we still need to move js file to the correct addon)